forge-game/src/main/java/forge/game/player/Player.java

```diff
@@ -303,6 +303,10 @@ public final PlayerCollection getOpponents() {
         return game.getPlayers().filter(PlayerPredicates.isOpponentOf(this));
     }
 
+    public final PlayerCollection getOpponentsInTurnOrder() {
+        return game.getPlayersInTurnOrder(this).filter(PlayerPredicates.isOpponentOf(this));
```

Comment:
did you test it?
looking at the code I'd expect it now respects ReverseTurnOrderEffect...
but that would still not return them based on where this player is seated 🤔

Reply:
Yes. It returns them in appropriate turn order. That said, I will do some more testing just to be sure. I'll amend this comment with results.

Reply:
See [this google doc](https://docs.google.com/document/d/15iNuRxOa5xcXwVdCdrEt2RZqIjsF3CTDyZfjUPOy0d0/edit?usp=sharing)

All seems to be good. I can test more if you have any suggestions of cases to test.

Reply:
yea seems fine, thanks